crates/storage/src/merkle/column.rs

```diff
@@ -31,6 +30,16 @@ pub enum MerkleizedColumn<TC> {
     MerkleMetadataColumn,
 }
 
+impl<TC> strum::EnumCount for MerkleizedColumn<TC>
+where
+    TC: strum::EnumCount + AsU32,
+{
+    /// The total count of variants in the enum.
+    /// Since we have two columns for each table column and one for the merkle data,
+    /// we have to multiply the count of the table columns by 2 and add one for the merkle metadata.
+    const COUNT: usize = TC::COUNT * 2 + 1;
+}
+
```

Comment:
What's the benefit of breaking this out?

Reply:
it's not a benefit, but the core part of the fix lol

Reply:
Oh I see from the description. we need the trait impl. Alright, fair enough 👍